src/web/app/page.component.ts

```diff
@@ -94,10 +89,6 @@ export class PageComponent implements OnInit {
     this.isCookieDisabled = !navigator.cookieEnabled;
   }
 
-  ngOnInit(): void {
-    this.statusMessageService.getAlertEvent().subscribe((result: StatusMessage) => {
-      this.messageList.push(result);
-    });
-  }
+  ngOnInit(): void { }
```

Comment:
Since we are not utilizing `ngOnInit()` at all, let us remove it. A [reference](https://stackoverflow.com/questions/35763730/difference-between-constructor-and-ngoninit) of the difference between `constructor` and `ngOnInit()`